2.2/root/opt/app-root/etc/puma.cfg

```diff
@@ -9,6 +9,11 @@ def get_max_memory()
     (2 ** (8*0.size - 2) - 1)
 end
 
+def per_worker_memory()
+    # Unless told otherwise, allow 256MiB per worker
+    ENV.fetch('MEMORY_BYTES_PER_WORKER', '268435456').to_i
```

Comment:
i'm not sure it really matters to have it as a constant elsewhere.  It's only used in one place and this one place is where the defaulting is done.  but up to you guys.

As for what the value should be...  we've already been slowly raising the value as we keep hitting users with applications for which we've selected too low of a default (or maybe it's php that we've been doing that for, but regardless, it is a recurring issue due to us trying to be too smart)

Given how terrible the experience is when we set this value too low(application crashloop), setting it aggressively high by default seems like the right thing to do.  Users who care about performance (and thus would not be happy with the conservative number of workers this gives them) need to be actively involved in tuning anyway for other reasons.